src/TagCollection.php

```diff
@@ -21,6 +21,28 @@ public function getAll()
         return $this->tags;
     }
 
+    public function getUntilMaxBytes($maxBytes=null)
```

Comment:
types, please

Reply:
Fixed in https://github.com/ostark/upper/pull/57/commits/b493328e9164b9c23ebb7165908d86dc0cb25dc9